app/iot/service.py

```diff
@@ -2,48 +2,47 @@
 import string
 from typing import Protocol
 
-from .message import Message, MessageType
+from app.iot.message import (Message, MessageType)
```

Comment:
Ensure that the `Message` and `MessageType` classes are correctly defined in the `app.iot.message` module. These imports are crucial for the `send_message` method and the `run_program` method in the `IOTService` class.